adapters/beachfront/beachfront.go

```diff
@@ -515,9 +533,7 @@ func (a *BeachfrontAdapter) MakeBids(internalRequest *openrtb2.BidRequest, exter
 
 	// For video, which uses RTB for the external request, this will unmarshal as expected. For banner, it will
 	// only get the User struct and everything else will be nil
-	if err := json.Unmarshal(externalRequest.Body, &xtrnal); err != nil {
-		errs = append(errs, err)
-	}
+	json.Unmarshal(externalRequest.Body, &xtrnal)
```

Comment:
Why are we ignoring the unmarshal error? As unlikely as it is, we believe it is better to check for it.

Reply:
I will stop ignoring it.